app/controllers/application_controller.rb

```diff
@@ -1,10 +1,15 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
   before_action :configure_permitted_parameters, if: :devise_controller?
+  before_action :create_cart
```

Comment:
Something about this doesn't feel right to me... Why do we want to call `create_cart` on literally every action we do in this app? What was the reason for moving it out of the ShoppingCartController?